README.md

```diff
@@ -4,7 +4,69 @@ bookend is a classifier for text data. We are building a machine learning model
 
 This project is part of The Erdos Institute's  [data science boot camp](https://www.erdosinstitute.org/code). Data sources include [Project Gutenberg](https://www.gutenberg.org/).
 
-## Getting started
+## Table of contents
+
+- [About the project](#about-the-project)
+  - [Methodology](#methodology)
```

Comment:
Is there an extra space here on purpose?

Reply:
The space doesn't make a difference in the formatting (in case you didn't know, if you go to "Files changed" and switch to the "rich diff" view, which will show the differences after HTML formatting)